internal/app/machined/pkg/controllers/runtime/events_sink.go

```diff
@@ -15,7 +16,6 @@ import (
 	"github.com/siderolabs/go-pointer"
 	"github.com/siderolabs/go-procfs/procfs"
 	"github.com/siderolabs/siderolink/api/events"
-	"go.uber.org/atomic"
```

Comment:
this looks werid

Reply:
It's actually not - I removed unnecessary side dependency since it's already in std.

Reply:
oh, I read it backwards :-D

I guess it was too late for me